analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -76,7 +76,9 @@ abstract class AbstractIntegrationSpec : StringSpec() {
     protected lateinit var downloadResult: Downloader.DownloadResult
 
     override fun beforeSpec(spec: Spec) {
-        outputDir = createTempDir(ORT_NAME, javaClass.simpleName)
+        // Do not use the usual simple class name as the suffix here to shorten the path which otherwise gets too long
```

Comment:
How many characters did the outputDir (as path string) actually have when it exceeded 256?
e.g. did it already exceed or did the exceed happen inside of `Download().download()`?

Reply:
It was something in the 26x range, I do not remember exactly, but removing the "SimpleFormIntegrationTest" suffix from the path really made the difference already here.

Reply:
Thanks!